src/main/java/ru/postgrespro/perf/pgmicrobench/statanalyzer/Multimodality/ModalityData.java

```diff
@@ -0,0 +1,40 @@
+package ru.postgrespro.perf.pgmicrobench.statanalyzer.Multimodality;
+
+import ru.postgrespro.perf.pgmicrobench.statanalyzer.Histograms.DensityHistogram;
+
+import java.util.List;
+
```

Comment:
предлагаю добавить lombok в зависимости и заменить все методы типа getSomething аннотацией Getter
```suggestion

@Getter
```